src/web/theme/pages/OurDemo/OurDemo.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Link from "theme/ui/atoms/Typography/Link";
+import "./OurDemo.scss";
+
+const OurProduct = [
```

Comment:
since it is not a component I would use snake case here : `ourProduct`, or even `OUR_PRODUCT`

Reply:
OOops